run_endpoint.sh

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env bash
+
+# Set up the routing needed for the simulation
+/setup.sh
+
+if [ -n "$TESTCASE" ]; then
+    case "$TESTCASE" in
+    # TODO: add supported test cases here
+    "versionnegotiation"|"handshake"|"transfer"|"retry"|"resumption"|\
+    "multiconnect"|"zerortt"|"chacha20")
+        ;;
+    *)
+        exit 127
+        ;;
+    esac
+fi
+
+# The following variables are available for use:
+# - ROLE contains the role of this execution context, client or server
+# - SERVER_PARAMS contains user-supplied command line parameters
+# - CLIENT_PARAMS contains user-supplied command line parameters
+
+if [ "$ROLE" == "client" ]; then
+    # Wait for the simulator to start up.
+    /wait-for-it.sh sim:57832 -s -t 30
+    cd /downloads || exit
+
+    # TODO: add client support
+    # I am not sure if the msquic codebase has an h09 client?
```

Comment:
The quicinterop tool is the client side tool that we use for `hq` testing. I'm not sure if it's generic enough for these purposes though.

Reply:
I never understood what options it takes, so someone who does would need to add support for it.